data/mods/Magiclysm/items/enchanted_melee.json

```diff
@@ -812,6 +812,27 @@
     "techniques": [ "WBLOCK_2", "RAPID", "SWEEP" ],
     "flags": [ "DURABLE_MELEE", "SHEATH_SPEAR", "TRADER_AVOID", "MAGIC_FOCUS" ]
   },
+  {
+    "id": "rune_magus_weapon_adept",
+    "type": "GENERIC",
+    "name": { "str": "Magus battle staff", "str_pl": "Magus battle staves" },
+    "//": "It will be based on the Q +2 weapon but sightly worse, sightly better than the magus staff, but it's greatest strength will be it's regen mana capabilities, better than magi staff minor but worse than the lesser one.",
```

Comment:
```suggestion
    "//": "It will be based on the Q +2 weapon but sightly worse, sightly better than the magus staff, but its greatest strength will be its regen mana capabilities, better than magi staff minor but worse than the lesser one.",
```